packages/ethtool/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=hhttps://www.kernel.org/pub/software/network/ethtool/
+TERMUX_PKG_DESCRIPTION="standard Linux utility for controlling network drivers and hardware, particularly for wired Ethernet devices"
+TERMUX_PKG_VERSION=4.11 
+TERMUX_PKG_SRCURL=https://www.kernel.org/pub/software/network/ethtool/ethtool-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=af2fd9692f3159d3ab1e41e6f9b7d8db2a4693f1cb22348c88ba89f70f0e6503
+TERMUX_PKG_MAINTAINER="Auxilus @Auxilus"
+TERMUX_PKG_DEPENDS="libnl, libnl-dev" 
+TERMUX_PKG_BUILD_IN_SRC=yes
+
+termux_step_post_make_install () {
+
+        mv $TERMUX_PREFIX/sbin/* $TERMUX_PREFIX/bin/
+        rm $TERMUX_PREFIX/sbin -rf
```

Comment:
You don't need to do this here - the main package already moves files from sbin/ to bin/, see

https://github.com/termux/termux-packages/blob/master/build-package.sh#L1007

And empty folders aren't kept. So you should be able to remove the whole `termux_step_post_make_install` block here!